dspace-api/src/main/java/org/dspace/util/SolrImportExport.java

```diff
@@ -166,7 +174,7 @@ else if ("export".equals(action))
                     {
                         String solrUrl = makeSolrUrl(indexName);
                         String timeField = makeTimeField(indexName);
-                        exportIndex(indexName, exportDir, solrUrl, timeField, lastValue);
+						exportIndex(indexName, exportDir, solrUrl, timeField, lastValue, line.hasOption(OVERWRITE_OPTION));
```

Comment:
@tdonohue , thanks for the alert.  My editor is set to insert spaces, so there is some other copy/paste magic that the IDE appears to be inserting.  I set my tab rendering to match GitHub's and that made it easier to track these down.  I will update the other recent PR's.